.travis.yml

```diff
@@ -29,5 +29,6 @@ branches:
   only:
     - master
     # bors branches
+    - powerset
```

Comment:
What's the reason for this?

Reply:
Great to hear, thanks! I'll remove that and push again, it was me just trying to get the CI building cleanly on my fork before updating this pull request (I forgot that the 'powerset' branch was still active for this pull request).

I've rewritten this feature and will post some further notes later on today.